oslabs-beta/Kubiquity
# Add CSV download

WHAT DOES THIS PR DO
1. Creates a Download component. 
2. Allows users to download K8s logs as a CSV file. 
3. Other various FE clean up tasks. 

HOW TO TEST THIS PR: 
1. Run `gh pr checkout 42 && npm install && npm run dev`
2. In the app, click the "Download" button, and verify that your data downloads as a CSV file.
